_config.php

```diff
@@ -31,8 +26,3 @@
     'sitetree_link',
     [SiteTree::class, 'link_shortcode_handler']
 );
-
-CMSMenu::remove_menu_class(CMSMain::class);
```

Comment:
Note that `CMSMain` itself is a special case and should _not_ get `ignore_menuitem` set to true.